### PR TITLE
Upload files for User 

### PR DESCRIPTION
**Pull request by AlonsoUriarte14**
Successfully implemented file upload (end to end) and  associate file to user account in database as well.

-added sql script to generate tables on backend so that developers can see the schemas i worked with

-Important note: this branch was merged with scholarshiptracker branch to get neccessary front end components. Also pulled from main to make sure everything works as it should.

### CODE REVIEW

File: database/test/tables.sql
Comment (Poggecci) [inline database/test/tables.sql:1]: Why are you manually creating tables? This is done by SQLAlchemy already. To ensure your table is being created, import  the SQLAlchemy model in the main.py

File: frontend/college-toolbox/src/components/ScholarshipCard.jsx [hunk 1]
```diff
@@ -67,6 +67,12 @@ const ScholarshipCard = ({
     fontSize: '0.9rem',
   };
 
+  // const userDocsRequest = {
```
Comment (Poggecci) [inline frontend/college-toolbox/src/components/ScholarshipCard.jsx:70]: remove commented code or add in brief justification explaining why it' there

File: backend/src/main.py [hunk 2]
```diff
@@ -122,7 +122,7 @@ def register_user(
 
 # Login endpoint
 @app.post("/login", response_model=LoginResponse)
-def login_user(
+async def login_user(
```
Comment (Poggecci) [inline backend/src/main.py:125]: Thank you for marking these as async, had slipped past me!

**Review by Poggecci — CHANGES_REQUESTED** [review]

**Comment by Poggecci** [comment]
Please add an adequate name for the Pull Request and link the relevant issues (you link stuff by clicking on the Development gear)
image

File: frontend/college-toolbox/src/components/ScholarshipCard.jsx (hunk 1 above)
Comment (AlonsoUriarte14) [reply frontend/college-toolbox/src/components/ScholarshipCard.jsx]: removed the commented code

File: database/test/tables.sql
Comment (AlonsoUriarte14) [reply database/test/tables.sql:1]: removed the sql since its unneccesarry

**Review by Poggecci — APPROVED** [review]

**Review by GabrielGranell15 — APPROVED** [review]

**Review by AcevedoC17 — APPROVED** [review]
Resolve conflict

**Review by Robert-6 — APPROVED** [review]

**Pull request merged**